static/js/request_row.js

```diff
@@ -36,7 +36,7 @@ $(document).ready(function() {
   $('.pending-details').each(function() {
     let that = $(this);
     let requestId = $(this).data('request');
-    $.getJSON('/api/requests/' + requestId + '/?exclude_canceled=true', function(data) {
+    $.getJSON('/api/requests/' + requestId + '/observations/?exclude_canceled=true', function(data) {
```

Comment:
I wonder how this got in here... this should have never worked with that URL

Reply:
Yeah, I guess it probably just slipped by unnoticed :woman_shrugging: